staging/src/k8s.io/kubelet/pkg/apis/podresources/v1alpha1/api.proto

```diff
@@ -18,6 +18,15 @@ option (gogoproto.goproto_unrecognized_all) = false;
 // node resources consumed by pods and containers on the node
 service PodResourcesLister {
     rpc List(ListPodResourcesRequest) returns (ListPodResourcesResponse) {}
+    rpc GetAllocatableResources(AllocatableResourcesRequest) returns (AllocatableResourcesResponse) {}
+}
+
+message AllocatableResourcesRequest {}
+
+// AllocatableResourcesResponses contains informations about all the devices known by the kubelet
+message AllocatableResourcesResponse {
+    repeated ContainerDevices devices = 1;
```

Comment:
The reuse of containerDevices here is kindof odd, I wonder if we should use a different structure name for it, especially if this is going to be used to sort devices by Numa nodes and by resource name

Reply:
The intention is to provide an output as much compatible as possible to the output of the other API calls. In this regard reusing some internal messages felt OK (same data). Open to suggestions.

Reply:
I pretty much expect this to return resources grouped by resourceName and that's it.
The fact that there might be multiple time the same resource name kinds of point to me that we either:
a) Don't want to group them by resource name
b) Want to have a different "explicit" structure, something that says that we have grouped devices by NUMA nodes.

Reply:
Now I fully understand your point. And that's a good point indeed. Let me think about the best approach.